apps/dashboard/config/configuration_singleton.rb

```diff
@@ -334,6 +334,19 @@ def allowlist_paths
     (ENV['OOD_ALLOWLIST_PATH'] || ENV['WHITELIST_PATH'] || "").split(':').map{ |s| Pathname.new(s) }
   end
 
+  # default value for opening apps in new window
+  # that is used if app's manifest doesn't specify
+  # if not set default is true
+  #
+  # @return [Boolean] true if by default open apps in new window
+  def open_apps_in_new_window?
+    if ENV['OOD_OPEN_APPS_IN_NEW_WINDOW']
```

Comment:
Do we want to add new ENV variables or instead start using the config files in ondemand.d/?  I know we're sort of in a transition, but I'm just wondering if we're introducing new config options if we should start pushing for the files instead?

Reply:
The config files are preferable but I would like to do a little more design before just adding new options there. So thought in the interim would add another env var.

Reply:
Works for me.